internal/effx/config.go

```diff
@@ -43,6 +44,13 @@ func DefaultConfigWithFlags() (*Configuration, []cli.Flag) {
 			Value:       cfg.APIKey,
 			EnvVars:     []string{"EFFX_API_KEY"},
 		},
+		&cli.StringFlag{
```

Comment:
I'd look at `cli.StringSliceFlag` which will handle repeated values for you.

Reply:
thanks, that makes it a little easier. done